app/assets/stylesheets/utilities/_typography.scss

```
@@ -99,3 +99,8 @@ h6,
 .h6 {
   @extend %h6;
 }
+
+.text-center--line {
+  border-bottom-color: color('primary-light');
+  line-height: 0.1em;
+}
```

Couple discrepancies I'm noticing between the implementation and Figma reference:

1. Font size: 22px vs. 18px in the Figma
2. Padding: 0.5rem vs. 1rem in the Figma

```suggestion
.text-center--line {
  border-bottom-color: color('primary-light');
  line-height: 0.1em;
  font-size: 1.125rem;

  span {
    @include u-padding-x(2);
  }
}
```